.github/workflows/agw-workflow.yml

```diff
@@ -454,10 +454,10 @@ jobs:
           clangFormatVersion: 11.1.0
           # taken from .clang-format
           style: file
-      - name: Check clang-format for lte/gateway/c
+      - name: Check clang-format for lte/gateway
         uses: DoozyX/clang-format-lint-action@v0.13
         with:
-          source: 'lte/gateway/c'
+          source: 'lte/gateway/c lte/gateway/python'
```

Comment:
I think that would work too. I just thought it would be better to be more explicit, so that it's clear that `lte/gateway` (without a `/c`) isn't a case of us not being specific enough. Otherwise someone might come along, assume the Python folder contains no C code and re-add the `/c` to the path.

Reply:
Good reasoning. :+1: